rebuild/src/secret_key.h

```diff
@@ -0,0 +1,28 @@
+#ifndef BLST_TS_SECRET_KEY_H__
+#define BLST_TS_SECRET_KEY_H__
+
+#include <memory>
+
+#include "addon.h"
+#include "blst.hpp"
+#include "napi.h"
+#include "public_key.h"
+#include "signature.h"
+
+#define BLST_TS_SECRET_KEY_LENGTH 32U
```

Comment:
In general, it's safer to use static variables than macros, which will give you proper typing.

Reply:
all macros were updated here 5d0225597d38fbb563ad64a6b2e908f817b2a906